api/ws/rpc/connect.js

```diff
@@ -48,25 +50,17 @@ const connectSteps = {
 			connectTimeout: TIMEOUT,
 			ackTimeout: TIMEOUT,
 			pingTimeout: TIMEOUT,
-			connectAttempts: 1,
 			port: peer.wsPort,
 			hostname: peer.ip,
 			query: System.getHeaders(),
-			multiplex: false,
+			multiplex: true,
```

Comment:
If I understand correctly, we want to reuse a socket instead of creating a second socket to the same address, so we `multiplex` is set to true? if its the case we can remove this option, as by default the `multiplex` is true.

Reply:
@ManuGowda Yes it's the default but it doesn't hurt to be explicit to avoid confusion. Also if there is an update to socketcluster-client in the future which changes the default, it's better if we lock it down.